loc/landlord_lookup.py

```diff
@@ -50,21 +52,22 @@ def _extract_landlord_info(json_blob: Any) -> Optional[LandlordInfo]:
     return None
 
 
-def lookup_landlord(address: str) -> Optional[LandlordInfo]:
-    '''
-    Looks up information about the landlord at the given address
-    and returns it, or None if no information could be gleaned.
-    '''
-
+def _lookup_bbl_and_full_address(address: str) -> Tuple[str, str]:
     features = geocoding.search(address)
     if not features:
-        return None
-    feature = features[0]
+        return ('', '')
+    props = features[0].properties
+    return (props.pad_bbl, props.label)
+
+
+def _lookup_landlord_via_network(pad_bbl: str) -> Optional[LandlordInfo]:
     url = settings.LANDLORD_LOOKUP_URL
+    if not url:
+        return None
```

Comment:
This is something I should have added a while ago; if the landlord lookup URL is empty, we don't want a bunch of errors from `requests` clogging up the logs/test output.